modules/common/__init__.py

```diff
@@ -23,7 +24,7 @@ def get_lines(input_filename):
         with gzip.open(input_filename, 'rb') as f:
             for i, l in enumerate(f):
                 pass
-    return i+1
```

Comment:
This method doesn't seem to do anything. It always returns 1.

Reply:
This method was used in a piece of code that it is gone. It counts the number of lines in a gzip file.
I will remove the method because it is not used anymore in the code.
I cleaned the code but it might be that there are other methods that are not used anymore.

Reply:
Cool. One of my favourite features of statically typed languages: the IDE can just remove all those things for you!

Reply:
Sure. I forgot to do it. :)